WebRTC-server/main.py

```
@@ -0,0 +1,175 @@
+import ssl
```

Ruff (0.12.2)

`ssl` imported but unused

Remove unused import: `ssl`

(F401)